src/config.rs

```diff
@@ -187,10 +187,8 @@ pub struct GooseConfiguration {
     #[options(no_short)]
     pub no_debug_body: bool,
     /// Tracks additional status code metrics
-    // Add a blank line and then an Advanced: header after this option
```

Comment:
Remove this and the next unnecessary change in this file.

Reply:
Done.